### DISCUSSION TITLE
Linkedin "Promoted"

### DISCUSSION THREAD

**Discussion by markinhfx** [uBlockOrigin/uAssets#20246] [Q&A]
In the last few days I am not getting bombarded with promoted ads in Linkedin.... I had been blocking them successfully with the following www.linkedin.com##[data-row]:has-text(Promoted) however it has stopped working.... see screenshot attached any suggestions greatly appreciated....
![image](https://github.com/uBlockOrigin/uAssets/assets/29015572/6b08232c-0b99-4e6a-874e-019cd7134cd2)

**Comment by mapx-** [comment]
try
`linkedin.com##span:has-text(Promoted):upward(.update-components-actor):upward(div)`

**Reply by markinhfx** [reply]
Thank You I will try that.

**Reply by Yuki2718** [reply]
Does `linkedin.com##.update-components-actor__description span[aria-hidden="true"] + span.visually-hidden:upward(.scaffold-finite-scroll__content > div)` work for you? It's independent to languages.

**Comment by intercalaris** [comment]
New filter that works on Ublock Origin

`http://linkedin.com/##div[role=%22listitem%22]:has(p:has-text(/^Promoted$/))`
